presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -93,37 +101,75 @@ Global Properties
 * ``cpuQuotaPeriod`` (optional): the period in which cpu quotas are enforced.
 
 Selectors are processed sequentially and the first one that matches will be used.
-In the example configuration below, there are five resource group templates.
-In the ``adhoc_${USER}`` group, ``${USER}`` will be expanded to the name of the
-user that submitted the query. ``${SOURCE}`` is also supported, which expands
-to the source submitting the query. The source name can be set as follows:
+
+Providing Selector Properties
+-----------------------------
+
+The source name can be set as follows:
 
   * CLI: use the ``--source`` option.
 
   * JDBC: set the ``ApplicationName`` client info property on the ``Connection`` instance.
 
-There are three selectors that define which queries run in which resource group:
+Client tags can be set as follows:
+
+  * CLI: use the ``--client-tags`` option.
+
+  * JDBC: set the ``ClientTags`` client info property on the ``Connection`` instance.
+
+Example
+-------
+
+In the example configuration below, there are several resource groups, some of which are templates.
+Templates allow administrators to construct resource group trees dynamically. For example, in
+the ``pipeline_${USER}`` group, ``${USER}`` will be expanded to the name of the user that submitted
+the query. ``${SOURCE}`` is also supported, which expands to the source submitting the query.
+You may also use custom named variables in source and user regular expressions.
 
-  * The first selector places queries from ``bob`` into the admin group.
+There are four selectors that define which queries run in which resource group:
 
-  * The second selector states that all data definition queries that come from a source that includes "pipeline"
-    should run in the user's personal data definition group, which belongs to the
-    ``globa.data_definition`` parent group.
+  * The first selector matches queries from ``bob`` and places them in the admin group.
 
-  * The third selector states that all queries that come from a source that includes "pipeline"
+  * The second selector matches all data definition queries from a source name that includes "pipeline"
+    and places them in the ``global.data_definition`` group. This could help reduce queue time for
+    this class of queries.
```

Comment:
Maybe add an explanation to the last sentence, along the lines of "since they are expected to be fast", that explains the motivation for this rule.